File: docs/docs/API-Reference/javascript-examples/api-openai-responses/retrieve-tool-call-results.js
const url = `http://${process.env.LANGFLOW_SERVER_URL ?? ""}/api/v1/responses`;

const options = {
  method: 'POST',
  headers: {
    "Content-Type": `application/json`,
    "x-api-key": `${process.env.LANGFLOW_API_KEY ?? ""}`,
  },
  body: JSON.stringify({
  "model": "FLOW_ID",
  "input": "Calculate 23 * 15 and show me the result",
  "stream": false,
  "include": [
    "tool_call.results"
  ]
}),
};

fetch(url, options)
  .then(async (response) => {
    if (!response.ok) {
      throw new Error(`HTTP ${response.status}`);
    }
    const text = await response.text();
    console.log(text);
  })
  .catch((error) => console.error(error));

File: docs/versioned_docs/version-1.9.0/API-Reference/python-examples/api-openai-responses/retrieve-tool-call-results.py
import os

import requests

base = os.environ.get("LANGFLOW_URL") or os.environ.get("LANGFLOW_SERVER_URL", "")
flow_id = os.environ.get("FLOW_ID", "")
api_key = os.environ.get("LANGFLOW_API_KEY", "")

url = f"{base}/api/v1/responses"

headers = {
    "Content-Type": "application/json",
    "x-api-key": api_key,
}

payload = {
    "model": flow_id,
    "input": "Calculate 23 * 15 and show me the result",
    "stream": False,
    "include": ["tool_call.results"],
}

response = requests.post(url, headers=headers, json=payload, timeout=120)
response.raise_for_status()

print(response.text)
